pretext/ExtraTopics/BinaryRepresentations.ptx

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-binaryrepresentations">
+  <title>Binary Representations</title>
+
+  <p>Have you ever seen all of the "hacker" 01010110101010s in the movies? As you probably
+    know, this is called binary. But it's not really how modern day hacking works. However,
```

Comment:
Maybe: Those 1's and 0's aren't actually how hacking works. However, binary is the base of all computing. Every word that you are reading right now was transmitted to your computer as a series of 1's and 0's. 

While nowadays you won't actually be writing binary at a keyboard, (unless you are a masochist), its still incredibly useful to know.